developers/agents/_includes/personalization_agent_tutorial_food_recommender.py

```diff
@@ -69,10 +69,9 @@
 )
 recipes_collection = client.collections.get("Recipes")
 
-with recipes_collection.batch.dynamic() as batch:
-    for i, item in enumerate(dataset):
-        if i < 200:
-            batch.add_object(properties=item["properties"])
+with recipes_collection.batch.fixed_size(batch_size=200) as batch:
+    for item in dataset:
```

Comment:
Heya, can we add the limit back in? I think I added it to speed up these examples.